extension-client/src/command/workspace/tfcost.ts

```diff
@@ -22,31 +22,45 @@ import * as util from '../../util';
 import EstimateCostView from '../../webview/workspace/EstimateCostView';
 
 export async function cost(context: vscode.ExtensionContext): Promise<void> {
+    let output = new util.OutputChannel();
+
     try {
+        output.appendLine('Verifying tfcost is installed');
         await command.tfcost.isInstalled();
         await util.workspace.createCredentialFile();
+        output.appendLine('Validating build');
         await command.terraform.init();
         await command.terraform.validate();
 
+        output.appendLine('Verifying tfvars file exists');
         var isTfvars = util.workspace.hasTfvarsFile();
         if (!isTfvars) {
             tfvarsFileNotFound();
+            output.fail('tfvars file does not exist');
             return;
         }
 
+        output.appendLine('Running terraform plan');
         const cred = await util.workspace.readCredentials();
         await command.terraform.createPlan(cred.apiKey);
+        output.appendLine('Creating planfile');
         await command.terraform.convertPlanToJSON(cred.apiKey);
+        output.appendLine('Calculating cost');
         await command.tfcost.calculateCost(cred.apiKey);
+        output.appendLine('Opening cost details in a web view');
         await new EstimateCostView(context).openView(false);
+        output.done();
     }
     catch (error: any) {
         if (error?.message.includes('command not found')) {
             tfcostNotFound();
+            output.fail('tfcost not found');
```

Comment:
We can output the original msg in this case instead of tfcost not found.
Since it can be any command eg:
`"Command failed: terraform init\n/bin/sh: terraform: command not found\n"`